.tours/-welcome-to-e-bikes.tour

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to E-Bikes",
+  "steps": [
+    {
+      "file": "force-app/main/default/lwc/productTileList/productTileList.js",
+      "description": "The Product Explorer lets you browse thought E-Bikes products by displaying cards that contain the product and it's price. It gets these by using an @wire to query call the getProducts Apex Class and providing the filter values that have been set. \n",
+      "line": 48
+    },
+    {
+      "file": "force-app/main/default/lwc/productCard/productCard.js",
+      "description": "The product card subscribes to the PRODUCT_SELECTED_MESSAGE message channel and passes the payload to the handleRecordLoaded method which will query for and return the selected record.",
+      "line": 57,
+      "selection": {
+        "start": {
+          "line": 57,
+          "character": 1
+        },
+        "end": {
+          "line": 57,
+          "character": 55
+        }
+      }
+    },
+    {
+      "file": "force-app/main/default/lwc/similarProducts/similarProducts.js",
+      "description": "The Similar Products component is accessed by a user on a record detail page. In LWC you can use the `@api recordId` property to gather the id from the pages context. ",
+      "line": 9
+    },
+    {
+      "file": "force-app/main/default/lwc/similarProducts/similarProducts.js",
+      "description": "Using Lightning Data Services, we can use an `@wire` to query for a records fields by passing in the `recordId` and the fields property that we have defined.",
```

Comment:
Add link to Lightning Data Services: https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.data_ui_api